lib/simple_segment/version.rb

```diff
@@ -1,3 +1,3 @@
 module SimpleSegment
-  VERSION = '0.1.1'
+  VERSION = '0.1.2'
```

Comment:
Could you undo this one? I'll bump the version separately when releasing to rubygems.